bridge/slack/slack.go

```diff
@@ -480,7 +485,7 @@ func (b *Bslack) handleMessageEvent(ev *slack.MessageEvent) (*config.Message, er
 	rmsg := config.Message{Text: ev.Text, Channel: channel.Name, Account: b.Account, ID: "slack " + ev.Timestamp, Extra: make(map[string][]interface{})}
 
 	// find the user id and name
-	if ev.BotID == "" && ev.SubType != messageDeleted && ev.SubType != "file_comment" {
+	if (ev.BotID == "" && ev.SubType != messageDeleted && ev.SubType != "file_comment") || (ev.BotID != "" && ev.Text == "" && ev.Attachments != nil) {
```

Comment:
I would change this line to
```
if ev.User != "" && ev.SubType != messageDeleted && ev.SubType != "file_comment" {
```
so that if we have a userID defined, it always checks the user